apisix/plugins/http-logger.lua

```diff
@@ -117,12 +139,52 @@ local function send_http_data(conf, log_message)
 end
 
 
-function _M.log(conf)
-    local entry = log_util.get_full_log(ngx, conf)
+local function gen_log_format(metadata)
+    local log_format = {}
+    if metadata == nil then
+        return log_format
+    end
+
+    for k, var_name in pairs(metadata.value.log_format) do
+        if var_name:sub(1, 1) == "$" then
```

Comment:
`var_name:byte(1, 1)` is already enough?

Reply:
thx, fixed